browser/ui/webui/playlist_ui.cc

```diff
@@ -25,6 +24,44 @@
 
 namespace playlist {
 
+namespace {
+
+class UntrustedPlayerUI : public ui::UntrustedWebUIController {
+ public:
+  explicit UntrustedPlayerUI(content::WebUI* web_ui)
+      : UntrustedWebUIController(web_ui) {
+    auto* source = CreateAndAddWebUIDataSource(
+        web_ui, kPlaylistPlayerURL, kPlaylistGenerated, kPlaylistGeneratedSize,
+        IDR_PLAYLIST_PLAYER_HTML);
+    source->AddFrameAncestor(GURL(kPlaylistURL));
+    source->OverrideContentSecurityPolicy(
+        network::mojom::CSPDirectiveName::ScriptSrc,
+        std::string("script-src 'self' chrome-untrusted://resources "
+                    "chrome-untrusted://brave-resources;"));
+    source->OverrideContentSecurityPolicy(
+        network::mojom::CSPDirectiveName::MediaSrc,
+        std::string("media-src 'self' chrome-untrusted://playlist-data "
+                    "https://*.googlevideo.com "
+                    "http://127.0.0.1:*;"));
```

Comment:
What is this for? If it's for tests, then we should be able to use port mapping and `https://*.googlevideo.com` URLs in tests instead.

Reply:
Exaclty. Changed to use host_resolver.